google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineStandardMavenLibrary.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.intellij.appengine.facet;
+
+import com.google.common.base.Predicate;
+import com.google.common.collect.Iterables;
+
+import com.intellij.openapi.roots.DependencyScope;
+
+import org.apache.commons.lang.WordUtils;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.idea.maven.utils.library.RepositoryLibraryProperties;
+import org.jetbrains.idea.maven.utils.library.RepositoryUtils;
+
+import java.util.Arrays;
+
+/**
+ * Defines the available App Engine standard maven-sources libraries.
+ */
+public enum AppEngineStandardMavenLibrary {
+  // TODO get display names from bundle (in form too)
+  SERVLET_API("Servlet API", new RepositoryLibraryProperties(
+      "javax.servlet", "servlet-api", "2.5"), DependencyScope.PROVIDED),
+  JSTL("JSP Standard Tag Library (JSTL)", new RepositoryLibraryProperties(
+      "javax.servlet", "jstl", RepositoryUtils.LatestVersionId), DependencyScope.PROVIDED),
+  APP_ENGINE_API("App Engine API", new RepositoryLibraryProperties(
+      "com.google.appengine", "appengine-api-1.0-sdk", RepositoryUtils.LatestVersionId),
```

Comment:
apparently we should be using RepositoryUtils.ReleaseVersionId so that we don't include non GA releases.  I'm a bit concerned as I've read some reports that any non fixed version number is not supported in Maven 3.